App/PyImports/StatusObjects.py

```diff
@@ -0,0 +1,115 @@
+import collections
+
+class StatusList(collections.MutableSet):
```

Comment:
It would be great to see why you decided to inherit from MutableSet. Please add a comment describing what the class is and what's it for.

Reply:
I decided to use MutableSet as it has the set properties and that we can add StatusItems as and when needed. Though this isn't used at the moment I envisaged that it could in the future.
I'm open to suggestions on other collection types to use as a base class.

Reply:
Adding documentation to `StatusObjects.py`